src/fpti.js

```diff
@@ -57,8 +57,10 @@ export const FPTI_KEY = {
     PAY_NOW:                        ('pay_now' : 'pay_now'),
     STICKINESS_ID:                  ('stickiness_id' : 'stickiness_id'),
     TIMESTAMP:                      ('t' : 't'),
+    TIME:                           ('time' : 'time'),
     OPTION_SELECTED:                ('optsel' : 'optsel'),
     USER_IDENTITY_METHOD:           ('user_identity_method' : 'user_identity_method'),
+    USER_ID:                        ('user_id' : 'user_id'),
```

Comment:
The `time` change makes sense and is well described. Why do we need this new `user_id` field? What will be passed into it?

Reply:
I'm fine with the change. I would just like to understand why we need it.

Reply:
Good point, I thought I could replace both of [these keys](https://github.com/paypal/paypal-smart-payment-buttons/blob/main/src/constants.js#L240), however they are amplitude keys, not fpti, so it is not needed anymore.. removing

Reply:
Regarding the necessity of the time key, Daniel pointed out that it is not needed either, [since it is set in](https://github.com/paypal/paypal-smart-payment-buttons/blob/ee31f1d4ee9bea646dcb21b5b7cf88fe4f8d974c/src/button/logger.js#L76) `src/button/logger.js`. So the problem is not here. Continue digging into it. I'll mark the PR  as draft meanwhile.